src/components/ApplicationForm/ReviewCards.js

```diff
@@ -0,0 +1,297 @@
+            <InfoGroup heading="Portfolio" data={formInputs.skills.portfolio} />
+            <InfoGroup heading="LinkedIn" data={formInputs.skills.linkedin} />
+            <InfoGroup heading="GitHub" data={formInputs.skills.github} />
+            <InfoGroup heading="Answer one of the two questions:" />
+            <InfoGroup heading="1. Describe how you became interested in the world of technology and where you hope to go from here on out!" />
+            <InfoGroup
+              heading="2. How would you like to challenge yourself during this hackathon?"
+              data={formInputs.skills.longAnswers}
+            />
+          </ContentWrapper>
+        </StyledBanner>
+      </ReviewContainer>
+
+      <ReviewContainer>
+        <JohnDiv>
+          <QuestionHeading>Almost there</QuestionHeading>
+          <Button
+            onClick={() => handleEdit('/application/part-3')}
+            height="short"
+            color="secondary"
+          >
+            Edit
+          </Button>
+        </JohnDiv>
+        <StyledBanner wide={true} blur>
+          <ContentWrapper>
+            {/* TODO: replace with actual values from formInputs */}
+            <InfoGroup
+              heading="You Heard about nwHacks From"
+              data={formInputs.questionnaire.engagementSource}
+            />
+            <InfoGroup heading="nwPlus Events Attended:" data={attendedVal} />
+          </ContentWrapper>
+        </StyledBanner>
+      </ReviewContainer>
+
+      <ReviewContainer>
+        <QuestionHeading>Terms &amp; conditions</QuestionHeading>
+        <ContentWrapper textBlock>
+          <P>
+            We participate in Major League Hacking (MLH) as a MLH Member Event. You authorize us to
+            share certain application/registration information for event administration, ranking,
+            MLH administration, and occasional messages about hackathons in line with the
+            <A bolded color="primary" src="https://mlh.io/privacy">
+              {' '}
+              MLH Privacy Policy
+            </A>
+            .
+          </P>
+        </ContentWrapper>
+        <ContentWrapper textBlock>
+          <P>
+            We also use your (anonymized!) data to help you get the best sponsors and continuously
+            improve nwHacks with each iteration. Our hackathon aims to connect you with industry
+            professionals, recruiters, and career opportunities. In doing so, information about our
+            hackers is needed in order for attending companies to contact you.
+          </P>
+        </ContentWrapper>
+        <ContentWrapper textBlock>
+          {/* TODO: replace termsAndConditions.MLHCodeOfConduct with formInputs.termsAndConditions.MLHCodeOfConduct for all termsAndConditions */}
+          <Checkbox
+            checked={termsAndConditions.MLHCodeOfConduct}
+            onChange={() =>
+              setTermsAndConditions({
+                ...termsAndConditions,
```

Comment:
you don't need to add this part as the merge function will handle it!

Reply:
ooh sorry not entirely sure what you mean? are you referring to not needing the spread operator "..." since I'm not sure how else I'd be able to preserve the state of the other checkboxes while flipping the current checkbox?